config/certs/DC=sbu, DC=state, CN=Configuration, CN=Services, CN=Public Key Services, CN=AIA, CN=U.S. Department of State AD Root CA 20200323.pem

```diff
@@ -0,0 +1,46 @@
+Subject: /DC=sbu/DC=state/CN=Configuration/CN=Services/CN=Public Key Services/CN=AIA/CN=U.S. Department of State AD Root CA
+Issuer: /C=US/O=U.S. Government/OU=FPKI/CN=Federal Common Policy CA
+-----BEGIN CERTIFICATE-----
+MIIHozCCBougAwIBAgICefkwDQYJKoZIhvcNAQELBQAwWTELMAkGA1UEBhMCVVMx
+GDAWBgNVBAoTD1UuUy4gR292ZXJubWVudDENMAsGA1UECxMERlBLSTEhMB8GA1UE
+AxMYRmVkZXJhbCBDb21tb24gUG9saWN5IENBMB4XDTE5MTIxODE3MDEyMVoXDTIy
+MTIxODE3MDAwNlowgbExEzARBgoJkiaJk/IsZAEZFgNzYnUxFTATBgoJkiaJk/Is
+ZAEZFgVzdGF0ZTEWMBQGA1UEAxMNQ29uZmlndXJhdGlvbjERMA8GA1UEAxMIU2Vy
+dmljZXMxHDAaBgNVBAMTE1B1YmxpYyBLZXkgU2VydmljZXMxDDAKBgNVBAMTA0FJ
+QTEsMCoGA1UEAxMjVS5TLiBEZXBhcnRtZW50IG9mIFN0YXRlIEFEIFJvb3QgQ0Ew
+ggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQDVZF3CZ2Jrfb6m0aOrxhfY
+XRlqaCOYbI6Mpyu2HXacbKW8Gtws3lsOzBxDRuPA/N4JYcRyC4nwYU91aGN1/7cZ
+rdtx5ngXyFHUzaegMmSSnW/zIS7b2d13E6ftWnUJwo3CgS4ngrKR2exABUPdIrQQ
+q8y53TzFuUVvajkFdQqf8tjHJBJ27g/KFYljcMzQuICemGfI1inGh5cTe3jVMb5W
+7BMreBfx7JuTybPZHdcwKK98TF2/X1i9ek9GNo43ytYAfdiLrZf2akOsUXs3Vn0e
+bTn7I6NdSBNrLg1xz7EHPAbKiQwt5CbPGWO4ZVU8x6JBU3QTPVAGVqeK1FnMpDtd
+gJ4/Bw+CHn74jDtn/TSaLnoV7IzCmZP0yey+zBCazTUVx5vuuVnBYla0kfVZ/3a5
+pTTeI6aHaAyQW1VhQ/mq2XUOLnR7FBUAW9vjFLkPglos0xAAlZR2fIMmcCiB+KaE
+8d+KvvUcINuxd25soTyRZcyZdlDwgn+L5tj59A2nOBDedpgKzoCKSDwaLMQRxw8y
+NshXjPpyLkw7V5AwtB9LEcxgBbU+S8G4wGo67I2PKLF/EPOGtFUt/QVB4o1q15VD
+enK/lFncKe/4dUR7q3sQhODP17EDE5WBfXwtrd3nf+bp8ftLeh/vjwuSIlzwWZ7/
+5RRCwEh7sMq64MqJ0i0GqQIDAQABo4IDGjCCAxYwDwYDVR0TAQH/BAUwAwEB/zBP
+BggrBgEFBQcBAQRDMEEwPwYIKwYBBQUHMAKGM2h0dHA6Ly9odHRwLmZwa2kuZ292
+L2ZjcGNhL2NhQ2VydHNJc3N1ZWRUb2ZjcGNhLnA3YzAPBgNVHSQECDAGgAEAgQEA
+MAoGA1UdNgQDAgEAMHkGA1UdIARyMHAwDAYKYIZIAWUDAgEDBjAMBgpghkgBZQMC
+AQMHMAwGCmCGSAFlAwIBAwgwDAYKYIZIAWUDAgEDJDAMBgpghkgBZQMCAQMNMAwG
+CmCGSAFlAwIBAxAwDAYKYIZIAWUDAgEDETAMBgpghkgBZQMCAQMnMFMGCCsGAQUF
+BwELBEcwRTBDBggrBgEFBQcwBYY3aHR0cDovL2NybHMucGtpLnN0YXRlLmdvdi9T
+SUEvQ2VydHNJc3N1ZWRCeUFEUm9vdENBLnA3YzCCAREGA1UdIQSCAQgwggEEMBgG
+CmCGSAFlAwIBAwYGCmCGSAFlAwIBBgMwGAYKYIZIAWUDAgEDBgYKYIZIAWUDAgED
+BjAYBgpghkgBZQMCAQMHBgpghkgBZQMCAQYMMBgGCmCGSAFlAwIBAwcGCmCGSAFl
+AwIBAwcwGAYKYIZIAWUDAgEDEAYKYIZIAWUDAgEGBDAYBgpghkgBZQMCAQMQBgpg
+hkgBZQMCAQMQMBgGCmCGSAFlAwIBAwgGCmCGSAFlAwIBBiUwGAYKYIZIAWUDAgED
+CAYKYIZIAWUDAgEDCDAYBgpghkgBZQMCAQMkBgpghkgBZQMCAQYmMBgGCmCGSAFl
+AwIBAyQGCmCGSAFlAwIBAyQwKQYDVR0eAQH/BB8wHaEbMBmkFzAVMRMwEQYKCZIm
+iZPyLGQBGRYDbWlsMA4GA1UdDwEB/wQEAwIBBjAfBgNVHSMEGDAWgBStDHp1XOXz
+mMR5mA6sKP2X9OcC/DA1BgNVHR8ELjAsMCqgKKAmhiRodHRwOi8vaHR0cC5mcGtp
+Lmdvdi9mY3BjYS9mY3BjYS5jcmwwHQYDVR0OBBYEFMwAaGGmpQOTEAobYbeHGMFF
+VtqCMA0GCSqGSIb3DQEBCwUAA4IBAQAPY+7dOPcDvedah6B3vy2xNeZFC5z0siy0
++LZUuzzAgb4AUA0cEksYmTYOhQsXnS+5xh+/i5Fl/NVzMzDjEgg2JdDUHHwqy9kv
+/4yqxSj5zSAxxvBSVZevMqX0tbm01NzQZY928IvSfH9+o+Sfxrgw4rIJw2n8Cpxl
+e0ojSMOyVT9cDFC9tBM2AhCudsE9THqVMqvaSTzuMIFGSnI5wbg4jrmXOya9Uqbg
+aoY2g7kVvGd4pCX0Bjd1Xd3Ofy6CikljpJmuzWr8s1fqP0Xe2LjNCjIWnglSx2Zp
+IT+C3MrEoJj/nbYyITsJecieAQJAYzVqGzDjz2OFo1ernfWirx2V
+-----END CERTIFICATE-----
+
```

Comment:
this cert seems to have an extra newline, but not all the others do? do we care about this?